Classes/PullRequestReviews/PullRequestReviewCommentsViewController.swift

```diff
@@ -200,4 +207,17 @@ final class PullRequestReviewCommentsViewController: MessageViewController,
         focusedReplyModel = reply
     }
 
+    // MARK: IssueCommentSectionControllerDelegate
+
+    func didSelectReply(to sectionController: IssueCommentSectionController, commentModel: IssueCommentModel) {
+        setMessageView(hidden: false, animated: true)
+        messageView.textView.becomeFirstResponder()
+        let quote = commentModel.rawMarkdown.substringUntilNewLine()
+        messageView.text = ">\(quote)\n\n@\(commentModel.details.login)"
```

Comment:
Can we make more of this shared in a common function? Especially since the logic is all basically the same.